package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/gwer/oncogene#readme",
   "devDependencies": {
     "babel-cli": "^6.23.0",
+    "babel-preset-babili": "0.0.12",
```

Comment:
Not node. NPM. In 3 version they are change a way for storage of modules. But you're right. It will be better, if you add `babel-preset-babili` and remove `babili`.